maintainers/maintainer-list.nix

```diff
@@ -4455,6 +4455,16 @@
     githubId = 1621335;
     name = "Andrew Trachenko";
   };
+  gordias = {
```

Comment:
If it's preferable to separate this out into its own PR, I'd be happy to do that.

Reply:
Another PR is not needed, but a different commit `maintainers: add gordias` would be appreciated

Reply:
Will do, thanks!